main.go

```diff
@@ -40,7 +40,11 @@ var migrations = []gomigrate.Migration{
 func GetIpfsDir() (string, error) {
 	ipfspath := os.Getenv("IPFS_PATH")
 	if ipfspath != "" {
-		return ipfspath, nil
+		expandedPath, err := homedir.Expand(ipfspath)
+		if err != nil {
+			return "", nil
```

Comment:
no error? This seems like an error.

Reply:
fixed. Although question, is it safe/smart to be using `homedir` this vendored way `homedir "github.com/ipfs/fs-repo-migrations/ipfs-2-to-3/Godeps/_workspace/src/github.com/mitchellh/go-homedir"`?

Reply:
It should be fine.